web/src/features/moreCast2/pages/MoreCast2Page.tsx

```diff
@@ -82,8 +83,10 @@ const MoreCast2Page = () => {
   const [modelType, setModelType] = useState<ModelType>(
     (localStorage.getItem(DEFAULT_MODEL_TYPE_KEY) as ModelType) || DEFAULT_MODEL_TYPE
   )
-  const [fromDate, setFromDate] = useState<DateTime>(DateTime.now())
-  const [toDate, setToDate] = useState<DateTime>(DateTime.now().plus({ days: 2 }))
+  const [fromTo, setFromTo] = useState<DateRange>({
+    startDate: DateTime.now().toJSDate(),
+    endDate: DateTime.now().plus({ days: 2 }).toJSDate()
```

Comment:
I think the off by one day bug is due to the initialization of `startDate` and `endDate` with `DateTime.now()`. `endDate` needs to be initialized exactly 2 days after `startDate` (to the millisecond).

Reply:
Wow, nice catch!